src/buffer/out/Row.cpp

```diff
@@ -957,9 +957,13 @@ til::CoordType ROW::MeasureLeft() const noexcept
     return gsl::narrow_cast<til::CoordType>(it - beg);
 }
 
-til::CoordType ROW::MeasureRight() const noexcept
+// Routine Description:
+// - Retrieves the position that is one after the last character in the row.
+// Arguments:
+// - useWrapness - If set and the row is wrapped, trailing whitespace is considered as a part of the text.
+til::CoordType ROW::MeasureRight(const bool useWrapness) const noexcept
 {
-    if (_wrapForced)
+    if (useWrapness && _wrapForced)

```

Comment:
That makes sense. I'll make the change.

Reply:
This has been changed. We do it in `ROW::GetLastNonSpaceColumn()` now.

Make sure you check my comment below about `_columnCount` usage in `row.MeasureRight()` because that affects both these functions.